statsmodels/tsa/arima_model.py

```diff
@@ -925,7 +927,10 @@ def fit(self, start_params=None, trend='c', method="css-mle",
             self.nobs = len(self.endog) - k_ar
 
         if start_params is not None:
-            start_params = np.asarray(start_params)
+            if hasattr(start_params, '__call__'):
```

Comment:
I don't find anything recent. pypi says 3.2 and 2.6 and 2.7 for statsmodels 0.6.1.
We never supported python 3.0 and most likely not 3.1 because at the time there were still too many changes.

Looks like we need to update the information in master and for 0.7.